packages/lexical-playground/src/plugins/FloatingLinkEditorPlugin/index.tsx

```diff
@@ -366,7 +366,7 @@ function useFloatingLinkEditorToolbar(
           if ($isRangeSelection(selection)) {
             const node = getSelectedNode(selection);
             const linkNode = $findMatchingParent(node, $isLinkNode);
-            if ($isLinkNode(linkNode) && (payload.metaKey || payload.ctrlKey)) {
+            if ($isLinkNode(linkNode) && (payload.altKey || payload.ctrlKey)) {
```

Comment:
It looks like this is the wrong event handler, this one is to open links (meta is correct for that)
```suggestion
            if ($isLinkNode(linkNode) && (payload.metaKey || payload.ctrlKey)) {
```